backend/app/api/v1/endpoints/employee_emergency_contacts.py

```
@@ -1,208 +1,157 @@
-async def create_emergency_contact(
+@router.post(
+    "/",
+    response_model=EmployeeEmergencyContactOut,
+    status_code=status.HTTP_201_CREATED,
+    summary="Create emergency contact",
+    description="Create an emergency contact for the current user."
+)
+async def create_emergency_contact_endpoint(
     contact: EmployeeEmergencyContactCreate,
+    request: Request,
+    current_user: Users = Depends(get_current_user),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)